.travis.yml

```diff
@@ -0,0 +1,40 @@
+language: ruby
+rvm:
+  - 2.6.3
+
+env:
+  global:
+    - GIT_USER="CI BOT"
+    - GIT_EMAIL="cibot@hudi.apache.org"
+    - GIT_REPO="apache"
+    - GIT_PROJECT="incubator-hudi"
+    - GIT_BRANCH="asf-site"
+    - DOCS_ROOT="`pwd`/docs"
+
+before_install:
+  - git config --global user.name ${GIT_USER}
+  - git config --global user.email ${GIT_EMAIL}
+  - git remote add hudi https://${GIT_TOKEN}@github.com/${GIT_REPO}/${GIT_PROJECT}.git
+  - git checkout -b pr
```

Comment:
is there any chance that this branch already exists?  (e.g travis caching the git clone etc?)

Reply:
We didn't specify the cache config like bellow, so IMO, the travis will not cache it.

```    
cache:
  directories:
    - "$HOME/.m2"
```

Reply:
okay.. just to be safe, do you want to account for that scenario? We can also file a follow up JIRA

Reply:
It's too late today, will double check it.

Reply:
Double checked, always say "Switched to a new branch 'pr'", means that travis doesn't cache it.

https://www.travis-ci.org/github/lamber-ken/hdocs/builds/664223886
https://www.travis-ci.org/github/lamber-ken/hdocs/builds/664223922
```
$ git checkout -b pr
Switched to a new branch 'pr'
```